spec/services/delete_work_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DeleteWork do
+  describe '.call' do
+    let!(:work) { create(:work, versions_count: 3, has_draft: true) }
+
+    before { class_double(SolrIndexingJob) }
+
+    specify do
+      expect(Work.count).to eq(1)
+      expect(WorkVersion.count).to eq(3)
+      described_class.call(work.uuid)
+      expect(SolrIndexingJob).not_to receive(:perform_later)
```

Comment:
I think, because this line is located after the `.call` that the rspec matcher needs to be `have_received` not `receive`

Reply:
I tried that, but I can't seem to get it to work with class doubles.

Reply:
If I change it do:

``` ruby
described_class.call(work.uuid)                                                                                      
expect(SolrIndexingJob).not_to have_received(:perform_later) 
```
I get:
```
Failure/Error: expect(SolrIndexingJob).not_to have_received(:perform_later)
       #<SolrIndexingJob (class)> expected to have received perform_later, but that object is not a spy or method has not been stubbed.
```

Reply:
Try using `class_spy`